resources/lib/model.py

```diff
@@ -189,10 +189,11 @@ def __init__(self, data: dict):
 
         meta = data.get("panel").get("episode_metadata")
 
-        self.title: str = meta.get("season_title") + " #" + meta.get("episode") + " - " + data.get("panel").get("title")
+        self.title: str = (str(meta.get("season_number")) + "x" if meta.get("season_number") else "") + utils.twoDigits(meta.get("episode_number")) + ". " + data.get("panel").get("title")
         self.tvshowtitle: str = meta.get("series_title", "")
         self.duration: int = int(meta.get("duration_ms", 0) / 1000)
         self.playhead: int = data.get("playhead", 0)
+        self.season: int = meta.get("season_number", "")
```

Comment:
Please add that to the `MovieData` object as well, with an empty string as a value. Otherwise it will trigger an error on movies.